src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/order/AddOrderCommandParserTest.java

```diff
@@ -132,20 +129,6 @@ public void parse_allFieldsPresent_success() {
                         + ORDER_PRICE_DESC_2 + DEADLINE_DESC_MARCH + ORDER_QUANTITY_DESC_1 + ORDER_QUANTITY_DESC_2,
                 new AddOrderCommand(expectedOrderDescriptorWithoutClientIndex));
 
-        //=========== Without Client Index and ingredient list ========================================================
```

Comment:
I don't know why these are removed by if you don't want these can you also remove the `expectedOrderDescriptorWithoutClientIndexAndIngredientList` and `expectedOrderDescriptorWithoutClientIndexAndOrderQuantity` above?